### DISCUSSION TITLE
Value in Changing the Python Code Right Now?

### DISCUSSION THREAD

**Discussion by mchoccie** [danielmiessler/Fabric#682] [Q&A]
So I know that the current implementation of Fabric is moving to Go. I want to make a few changes to the existing Python code and add it as a contribution but I was wondering if it made any sense to do that given we're shifting to Go. Let me know what you guys think.

**Comment by AO12612** [comment]
go version should be really close:
https://github.com/danielmiessler/fabric/discussions/467#discussioncomment-9897216
